docs/91_version_history.rst

```diff
@@ -25,6 +25,71 @@ Version History
+       - A number of optimisations have been made to improve the performance of Python filters within a pipeline.
+       - The support bundle has been updated to include a list of the Python packages installed on the machine.
+       - The configuration category C++ API has been enhanced in the retrieval and setting of all the attributes of a configuration item.
+       - Changes have been made to the purge process in the sqlitelb and sqliteinmemory plugins in order to improve performance.
+       - A change has been made to the configuration of the storage plugin such that rather than having to type correct names for storage plugins the user may now select the plugins to use from a drop down list. Note however that the system must still be restarted for the new storage plugin to take effect.
+       - The storage service has been updated to allow other services to subscribe the notifications of inserts into the generic tables.
+       - The documentation regarding handling and updating certificates used for authentication has been updated.
+
+
+    - Bug Fix:
+
+       - An issue that caused the retrieval of system logs for services with white space in the name of the service has been resolved.
+       - An erroneous message was being produced when starting the system using the SQLite in memory storage plugin. This has now been resolved.
+       - Support has been improved for switching between different storage plugins that allows for correct schema creation when using different sqlite plugin variants for configuration and readings storage.
+
+
+- **GUI**
+
+    - Bug Fix:
+
+       - An issue that caused services with quotes in the name to disappear from the user interface has been resolved.
+       - An issue with the handling of script type items whose name was not also script in the user interface that meant that scripts with different names were incorrectly handled has been resolved.
+       - An issue with editing large scripts or JSON items in the configuration has been resolved.
+       - An occasional error that appeared on the Control Script and ACL pages has been resolved.
+       - A text wrapping issue in the system log viewer has been resolved.
+
+
+- **Services & Plugins**
+
+    - New Features:
+
+       - A new south plugin, fledge-south-J1939, has been created to collect data into Fledge.
+       - Support for multiple data centers for OSIsoft Cloud Services (OCS) has been added. OCS is hosted in the US-West and EU-West regions.
+       - There have been improvements to the OMF north plugin to prevent an issue that could cause the plugging to stop sending data if the type of an individual datapoint changed repeatedly between integer and floating point values. The logging of the the plugin has also been improved, with clearing messages and less repetition of error conditions that persist for long periods.
```

Comment:
plugging?

Reply:
The logging of the the plugin -> The logging of the plugin

BTW plugin word used too many times